docs/source/guide/tutorials/base/how_to_train/action_detection.rst

```diff
@@ -0,0 +1,145 @@
+
+.. code-block::
+
+    training_extensions
+    └── data
+        └── JHMDB_5%
+            ├── train
+            │    └── brush_hair_Brushing_Hair_with_Beth_brush_hair_h_nm_np1_le_goo_0
+            │        ├── annotations.xml
+            │        └── images [40 frames]
+            │
+            │── test
+            │    └── brush_hair_Aussie_Brunette_Brushing_Long_Hair_brush_hair_u_nm_np1_fr_med_0
+            │        ├── annotations.xml
+            │        └── images [40 frames]
+            │
+            │── train.pkl
+            └── test.pkl
+
+
+*********
+Training
+*********
+
+1. First of all, we need to choose which action detection model will we train.
+The list of supported templates for action detection is available with the command line below.
+
+.. note::
+
+  The characteristics and detailed comparison of the models could be found in :doc:`Explanation section <../../../explanation/algorithms/action/action_detection>`.
+
+.. code-block::
+
+  (otx) ...$ otx find --task action_detection
+
++------------------+---------------------------------------+---------------+---------------------------------------------------------------------+
+|       TASK       |                   ID                  |      NAME     |                              BASE PATH                              |
++------------------+---------------------------------------+---------------+---------------------------------------------------------------------+
+| ACTION_DETECTION | Custom_Action_Detection_X3D_FAST_RCNN | X3D_FAST_RCNN | otx/algorithms/action/configs/detection/x3d_fast_rcnn/template.yaml |
++------------------+---------------------------------------+---------------+---------------------------------------------------------------------+
+
+To have a specific example in this tutorial, all commands will be run on the X3D_FAST_RCNN  model. It's a light model, that achieves competitive accuracy while keeping the inference fast.
+
+2. Next, we need to create workspace
+for various tasks we provide.
+
+Let's prepare an OpenVINO™ Training Extensions action detection workspase running the following command:
+
+.. code-block::
+
+  (otx) ...$ otx build --train-data-roots ./data/JHMDB_5%/train --val-data-roots ./data/JHMDB_5%/test --model X3D_FAST_RCNN
+
+  [*] Workspace Path: otx-workspace-ACTION_DETECTION
+  [*] Load Model Template ID: Custom_Action_Detection_X3D_FAST_RCNN
+  [*] Load Model Name: X3D_FAST_RCNN
+  [*]     - Updated: otx-workspace-ACTION_DETECTION/model.py
+  [*]     - Updated: otx-workspace-ACTION_DETECTION/data_pipeline.py
+  [*] Update data configuration file to: otx-workspace-ACTION_DETECTION/data.yaml
+
+  (otx) ...$ cd ./otx-workspace-ACTION_DETECTION
+
+It will create **otx-workspace-ACTION_DETECTION** with all necessery configs for X3D_FAST_RCNN, prepared ``data.yaml`` to simplify CLI commands launch and splitted dataset.
```

Comment:
```suggestion
It will create **otx-workspace-ACTION_DETECTION** with all necessary configs for X3D_FAST_RCNN, prepared ``data.yaml`` to simplify CLI commands launch and splitted dataset.
```